LoopBack/LBUser.m

```diff
@@ -91,9 +117,35 @@ - (void)logoutWithSuccess:(LBUserLogoutSuccessBlock)success
                      success:^(id value) {
                          LBRESTAdapter* adapter = (LBRESTAdapter*)self.adapter;
                          adapter.accessToken = nil;
+                         self.currentUserId = nil;
```

Comment:
Oh, I see, you are clearing `cachedCurrentUser` whenever `currentUserId` is changed. Ignore my previous comment then.